README.md

```diff
@@ -93,6 +93,9 @@ If you want to run all the tests from source, do:
 Building is not required. You are highly recommended to download the releases
 we provide from the Logstash site!
 
+**Note** Before you build the artifacts, you need to run:
+    rake artifact:freeze-defaults-gemfile 
```

Comment:
I think this needs a blank line before the `rake` stuff for it to render correctly.